src/components/dialog-full-screen/dialog-full-screen.stories.mdx

```diff
@@ -83,6 +83,44 @@ const MyComponent = ({ isOpen, handleClose }) => (
   </Story>
 </Preview>
 
+### With disableContentPadding
+
+<Preview>
+  <Story name="with disableContentPadding" parameters={{ chromatic: { disable: true }}}>
```

Comment:
I think we want to enable this chromatic snapshot

Reply:
the `open` state is false so chromatic will do a snapshot to the opening button, not to the Dialog. So we won't be able to see  the padding. This is why I left it as disable